etc/kubernetes/helm/enterprise-gateway/templates/ingress.yaml

```diff
@@ -16,6 +16,9 @@ metadata:
 {{ toYaml .Values.ingress.annotations | indent 4}}
 {{- end }}
 spec:
+{{ if .Values.ingress.ingressClassName }}
```

Comment:
Should we define an empty placeholder, similar to [`kernelspecs.image`](https://github.com/jupyter-server/enterprise_gateway/blob/2f9d635f9d9d85ae386b40f7b222b45ca580f978/etc/kubernetes/helm/enterprise-gateway/values.yaml#L96), so that folks have an idea that this is configurable?

Reply:
> Would you regards a YAML schema important, so that each field and its types/ values can become self-documented and easily validated ?

That sounds great! I'd like us to leverage schemas/correctness as much as possible.